src/crimson/os/cyan_store.h

```diff
@@ -4,17 +4,55 @@
 #pragma once
 
 #include <string>
+#include <unordered_map>
+#include <map>
+#include <vector>
+#include <seastar/core/future.hh>
+#include "osd/osd_types.h"
 #include "include/uuid.h"
 
+namespace ceph::os {
+
+class Collection;
+class Transaction;
+
```

Comment:
we remove the objectstore class, is it benefit for integrating blue-store and other stores?

Reply:
@liu-chunmei eliminating `ObjectStore` is merely a side effect. i am trying to get a working "MemStore" for crimson-osd with minimal efforts. in future, we will replace `CyanStore` with a futurized wrapper around `ObjectStore` or `BlueStore`.